### DISCUSSION TITLE
MicroSD

### DISCUSSION THREAD

**Discussion by Ceilog** [thehighnibble/thehighnibble.github.io#132] [IMSAI8080esp replica]
![20240111_154345](https://github.com/thehighnibble/thehighnibble.github.io/assets/146817561/b4e4d8ed-96da-4e10-98c2-e55a323c5a44)

I have some trouble with step 8.
Cant read the SD in this test.

I have good connection from every point.
And no crossconections. But I notice that third test point also has a connection to ground. The one in the picture. Should it be so? If I look at other side of PCB I guess its the groundplane?

**Comment by nbreeden2** [comment]
Where you have the probe is uSD pin 6 which is VSS/Ground

Can you verify you checked each set of pins for shorts (1-2, 2-3, 3-4 ....).

There may be a solder bridge from pin 8 to VSS.

<img width="937" alt="image" src="https://github.com/thehighnibble/thehighnibble.github.io/assets/37715892/163f7f24-0359-47a3-8aba-0f5a0c24e768">
